db/migrate/20231016000000_introduce_host_products_report.rb

```diff
@@ -0,0 +1,10 @@
+class IntroduceHostProductsAndSubscriptionsReport < ActiveRecord::Migration[6.1]
+  def up
+    token = SecureRandom.base64(5)
+    ReportTemplate.unscoped.find_by(name: "Host - Products")&.update_columns(:name => "Host - Products #{token}")
+  end
+
+  def down
+    ReportTemplate.unscoped.find_by(name: "Host - Products")&.delete
+  end
```

Comment:
I thought these migrations were only needed when renaming a report? What does it do for us when adding a new report?

Reply:
If a user already has a report named `Host - Products`, it will be overwritten otherwise.